### ISSUE TITLE
sphinx-quickstart: error: the following arguments are required: PROJECT_DIR

### ISSUE THREAD

**Issue by agzimmerman** [sphinx-doc/sphinx#4617]
Subject: sphinx-quickstart invocation doc seems incorrect.

### Problem
I am trying to use Sphinx (version 1.7) for the first time. I am following the [First steps guide](http://www.sphinx-doc.org/en/1.7/tutorial.html) and the [ReadTheDocs Getting Started guide](https://docs.readthedocs.io/en/latest/getting_started.html), both of which say to enter only 
`sphinx-quickstart`

But this returns
> usage: sphinx-quickstart [OPTIONS] <PROJECT_DIR>
> sphinx-quickstart: error: the following arguments are required: PROJECT_DIR

[The docs](http://www.sphinx-doc.org/en/stable/invocation.html) say "If you omit projectdir, files are generated into current directory by default."

Side note: As an amateur Python developer using Sphinx for the first time, my first assumption would have been that "PROJECT_DIR" would refer to the root directory of my Python project.

#### Procedure to reproduce the problem
```
sphinx-quickstart
```

#### Error logs / results
```
usage: sphinx-quickstart [OPTIONS] <PROJECT_DIR>
sphinx-quickstart: error: the following arguments are required: PROJECT_DIR
```
http://www.sphinx-doc.org/en/stable/invocation.html

#### Expected results
sphinx-quickstart should execute, and begin prompting the user for manual inputs related to various options. See http://www.sphinx-doc.org/en/stable/invocation.html

### Reproducible project / your project
https://hub.docker.com/r/zimmerman/phaseflow-fenics/

### Environment info
- OS: Linux 818a60beed73 4.9.60-linuxkit-aufs #1 SMP Mon Nov 6 16:00:12 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
- Python version: Python 3.5.2 (default, Sep 14 2017, 22:51:06) [GCC 5.4.0 20160609] on linux
- Sphinx version: 1.7.0

**Comment by tk0miya** [comment]
Sorry for inconvenience. It's a bug of quickstart command.
I just made a bug fix #4621.

Please give path as an argument until fixed.
Thanks,

**Comment by tk0miya** [comment]
Fixed by #4621 
Thank you for reporting!
